.github/workflows/wheels_upload_pip.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         shell: bash
     steps:
       - name: Recursive checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
Is this necessary?

Reply:
Why not?

Reply:
in some docker images there is libc mismatch between one available in the docker and one required by nodejs
you'll see error messages during the workflow if this is the case
(like this https://github.com/actions/checkout/issues/1590)